raymon/profiling/extractors/vision/anomaly.py

```diff
@@ -1,70 +1,84 @@
+# https://github.com/onnx/tutorials#converting-to-onnx-format
+# https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html
+
+import onnxruntime
+import pkg_resources
+
 import numpy as np
-import torch
 import base64
-import torchvision.models as models
-from torch.utils.data import DataLoader, Dataset
-from torchvision import transforms
 from PIL import Image
+from collections.abc import Iterable
 
 from raymon.profiling.extractors.structured.kmeans import KMeansOutlierScorer
 
-# Loosely based on "Deep Nearest Neighbor Anomaly Detection": https://arxiv.org/abs/2002.10445
-
-
-class ImageDataset(Dataset):
-    """Face Landmarks dataset."""
-
-    def __init__(self, loaded_data, transform=None):
-
-        self.loaded_data = loaded_data
-        self.transform = transform
-
-    def __len__(self):
-        return len(self.loaded_data)
-
-    def __getitem__(self, idx):
-        sample = self.loaded_data[idx]
-        if self.transform:
-            sample = self.transform(sample)
-        return sample
+model_path = pkg_resources.resource_filename("raymon", "models/mobilenetv2-7.onnx")
```

Comment:
Do not execute code on import in normal python modules. Move this inside the function where you need it.